mod_reforged/hooks/config/strings.nut

```diff
@@ -860,6 +861,15 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
 	}),
+	RF_CalculatedStrikes = ::UPD.getDescription({
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Deal " + ::MSU.Text.colorGreen("20%") + " more Damage against anyone who has not started their [turn|Concept.Turn] yet in the current [round|Concept.Round]."
+				::MSU.Text.colorRed("20%") + " less [Initiative|Concept.Initiative]."
```

Comment:
I don't think this looks good in perk descriptions. We should instead say `Initiative is reduced by 20%`.  This `20% less initiative` style should be used in status effect tooltips instead.

Reply:
How about `You have 20% less Initiative`?

Reply:
Not good. That sounds like a descriptive statement of state rather than description of what would happen as a result of this perk i.e. it sounds like I am telling someone what they already have rather than what would happen to them. I don't think we need to shoehorn the less/more wording into perk descriptions. The original way of saying it reads much better and is more in line with the game's style in my opinion.